modules/my_enumerable.rb

```diff
@@ -0,0 +1,19 @@
+module MyEnumerable
+  def all?
+    arr = []
+    list.each { |el| arr.push(el) if yield el }
+    puts arr.length == list.length
+  end
+
+  def any?
+    arr = []
+    list.each { |el| arr.push(el) if yield el }
+    puts !arr.empty?
+  end
+
+  def filter
+    arr = []
+    list.each { |el| arr.push(el) if yield el }
+    puts arr
+  end
```

Comment:
- [ ] This filter method is supposed to print numbers that are `even` for this case it should print `[2, 4]` please work on that

Reply:
@DeliceLydia Sure will do. Thanks for the review